vscode-cairo/README.md

```diff
@@ -48,3 +48,13 @@ If successful, go back to `sudo npm install -g vsce` and continue from there.
    ```
 3. Reload VSCode.
 4. Press F5.
+
+# Logging
+
+If you want the logs to be printed in your VSCode terminal when you click “Debug”/”Run” from the CodeLens above the test, add this to your ~/.config/Code/User/settings.json:
+```
+"rust-analyzer.runnableEnv": {
+   "RUST_LOG": "debug,salsa=off,minilp=off"
```

Comment:
Is this the place to add the CARGO_MANIFEST thingy to make debug from vscode work? If so, maybe write it as well?